core/ledger/kvledger/txmgmt/statedb/statedb.go

```diff
@@ -81,6 +81,8 @@ type VersionedDB interface {
 	Open() error
 	// Close closes the db
 	Close()
+	// Remove removes the db data for the channel associated to the versionedDB
+	Remove() error
```

Comment:
See comment on couchdb impl regarding having this function here v/s on the provider